arch/sim/src/sim/up_internal.h

```diff
@@ -242,6 +242,7 @@ void sim_cpu0_start(void);
 void up_cpu_started(void);
 int up_cpu_paused(int cpu);
 struct tcb_s *up_this_task(void);
+int host_setcpupause_request(int irq);
```

Comment:
change to up_cpu_set_pause_handler?

Reply:
I updated the name to `up_cpu_set_pause_handler `